frontend/tickets-online/src/routes.js

```diff
@@ -1,27 +1,47 @@
-import React from 'react';
-import { Switch, withRouter } from 'react-router-dom';
-import { connect } from 'react-redux';
+import React from "react";
```

Comment:
As I saw you are using single quotes. I also recommend to use signle. Do you review files that you are going to commit?